Tealium/src/ios/TealiumPlugin.swift

```diff
@@ -103,6 +103,15 @@ class TealiumPlugin: NSObject {
         tealium?.dataLayer.all[key]
     }
 
+    @objc
+    public static func gatherTrackData(callbackId: String) {
+        tealium?.gatherTrackData(completion: { trackData in
+            let result = CDVPluginResult(status: CDVCommandStatus_OK, messageAs: trackData)
+            result?.keepCallback = true
```

Comment:
same comment as above, re. `keepCallback`

I'd also suggest that this method returns the value from the call to `gatherTrackData` - at the moment, both the `gatherTrackData` methods in `TealiumPlugin` and `TealiumCordova` seems to handle the callback... making one of them superfluous.